packages/cli/src/tools/ios/findPodspecName.js

```diff
@@ -29,5 +29,5 @@ export default function findPodspecName(folder) {
     }
   }
 
-  return podspecFile.replace('.podspec', '');
+  return podspecFile;
```

Comment:
I think this might be classed as a breaking change for the current `link` command

Reply:
Ah yes, good call 👍